lib/redux/middleware/navigation_middleware.dart

```diff
@@ -13,6 +14,9 @@ class NavigationMiddleware extends MiddlewareClass<AppState>{
     if(action is GoToCropDetailAction){
       Keys.navKey.currentState.pushNamed(AppRoutes.cropDetail);
     }
+    if(action is GoToArticleDetailAction) {
+      Keys.navKey.currentState.pushNamed(AppRoutes.articleDetail);
```

Comment:
maybe a switch case is better for this class?

Reply:
with switch isn't working for the moment. The middleware seems do not support switch cases yet.